### PR TITLE
Better README. Renamed Env Variables to be more specific

### PR DESCRIPTION
**Pull request by cdagli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+
+
+**CLI Usage:**
+
+````
+Usage: index [options]
+
+  Options:
+
+    --chrome-version [chromeVersion]                        Chrome browser major version string e.g. 65
```
Comment (thehcma) [inline README.md]: ..., e.g.,... (the comma pre/post is a weird English grammar rule)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+    --chrome-driver-target-path [chromeDriverTargetPath]    Path to install Chrome driver executable
+    --firefox-version [firefoxVersion]                      Firefox browser major version string e.g. 57
+    --firefox-driver-target-path [firefoxDriverTargetPath]  Path to install Firefox driver(geckoDriver) executable
```
Comment (thehcma) [inline README.md]: space after driver 
looks like the preferred spelling for the name of the driver is _geckodriver_

File: installer.spec.js [hunk 3]
```diff
@@ -34,43 +37,44 @@ describe('browserDriverInstaller', function ()
...
     {
         const wrongVersionNumber = '1';
-        expect(function () { installer.driverInstaller(wrongVersionNumber, '/some/target/path'); }).to.throw(
-            /Failed to locate a version of ChromeDriver that matches the installed version of Chrome \(1\). Valid Chrome versions are:*/
+        expect(function ()
+        {
+            installer.driverInstaller('chrome', wrongVersionNumber,
+                '/some/target/path');
+        }).to.throw(
+            /Failed to locate a version of chromedriver that matches the installed version of chrome \(1\). Valid chrome versions are:*/
```
Comment (thehcma) [inline installer.spec.js:52]: Chrome

File: .travis.yml [hunk 4]
```diff
@@ -1,3 +1,9 @@
 language: node_js
 node_js: 
-  - "node"
\ No newline at end of file
+  - "node"
+env: 
+  global: 
+    - BROWSER_DRIVER_INSTALLER_CHROME_VERSION=67
```
Comment (thehcma) [inline .travis.yml:6]: ideally, we could have an alias for the latest version, so we don't need to hard code this here (assuming, that is, this is possible) and we wouldn't have to change the versions here also

File: README.md [hunk 5]
```diff
@@ -1,4 +1,32 @@
...
+
+
+**CLI Usage:**
+
+````
+ Usage: index [options]
+
+  Options:
+
+    --browser-name <browserName>        Browser to install the driver for
```
Comment (thehcma) [inline README.md:25]: in our command-line tools we do: "sets <blah> <blah>" and -h "outputs..."

**Review by thehcma — APPROVED** [review]

File: .travis.yml (hunk 4 above)
Comment (cdagli) [reply .travis.yml:6]: I've created #3 and #4

**Pull request merged**